src/main/kotlin/racingcar/Driver.kt

```diff
@@ -1,12 +1,23 @@
 package racingcar
 
-class Driver(private val car: Car) {
-    val driveRecord = mutableListOf<Int>()
+class Driver(val name: String, private val car: Car) {
+
+    val driveRecord = mutableListOf<Int>(car.movement)
+
+    init {
+        require(name.trim().length in (1..5)) {
```

Comment:
`MOVE_RANGE` 를 상수로 추출한 것 처럼 이곳도 상수 추출을 해보는건 어떨까요?
```suggestion
        require(name.trim().length in CONTANT) {
```